FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 11.0.0
+- [removed] **Breaking change**: The deprecated `FirebaseInAppMessagingSwift`
+  module has been removed. See
+  https://firebase.google.com/docs/ios/swift-migration for migration
+  instructions.
+
```

Comment:
@paulb777 @morganchen12, for the Swift extension removal, WDYT of this message? I can add one to the main product changelog for each affected product.  Alternatively (or additionally), I could one in the main Firebase changelog.

I think this message in each of the affected products is fine.

Reply:
I like it, as well as making a generic overall message that all Swift extensions are now merged